src/components/calcite-link/calcite-link.tsx

```diff
@@ -110,7 +111,29 @@ export class CalciteLink {
 
   @Method()
   async setFocus(): Promise<void> {
-    this.childEl.focus();
+    focusElement(this.childEl);
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  //--------------------------------------------------------------------------
+
+  @Listen("click")
+  protected handleClick(event: KeyboardEvent | MouseEvent): void {
```

Comment:
IMO it seems better to just have the click event on the childEl instead of having to capture it on the parent and then programmatically call it on the child.

Is there not a way to fix the screen reader without doing this? It just seems a little hacky :)

Reply:
This PR tries to prevent interaction with the internal anchor to prevent focusing on it after being activated. I'll see if I can make it less hacky.